src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Helpers/PlatformHttpException.cs

```diff
@@ -32,25 +32,10 @@ private static string ModifyMessage(HttpResponseMessage response)
             return message;
         }
 
-        /// <summary>
-        /// Just a single line
-        /// </summary>
-        /// <param name="message">the message</param>
-        public PlatformClientException(string message) : base(message)
-        {
-        }
-
-        /// <summary>
-        /// Inner exception
-        /// </summary>
-        public PlatformClientException(string message, Exception innerException) : base(message, innerException)
-        {
-        }
-
         /// <summary>
         /// C# stuff to what knows why
```

Comment:
Not C#, but .Net. It is good practice when creating a custom Exception with a new property. (HttpResponseMessage)